docs/discover/field-statistics.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[show-field-statistics]]
+== Explore the fields in your data
+
+beta::[]
```

Comment:
@gchaps this reminds me of ... we should also add document explorer, it would also gain visibility by adding a beta entry in our documentation